test/integration/Client/client_lib_metadata_rpc_test.cpp

```diff
@@ -58,7 +58,7 @@ int main() {
         t1 = std::chrono::steady_clock::now();
         ret = client.EditChronicleAttr(chronicle_name, key, "2023-01-15");
         t2 = std::chrono::steady_clock::now();
-        assert(ret == CL_SUCCESS);
+        //assert(ret == CL_SUCCESS);
```

Comment:
Why do you comment this assert statement? Is it not working? Same for the next assert for GetChronicleAttr().